python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -797,6 +797,40 @@ def get_hashable(self) -> Hashable:
             json.dumps(self.cloud_options),
         )  # pragma: no cover
 
+    @classmethod
+    def _write_csv(
```

Comment:
Thanks for the review @TomAugspurger and @Matt711 - Just wanted to point out that I did a bit of refactoring here **after** you approved. Let me know if I should revert/adjust anything.

Reply:
Looks good, thanks. (I guess technically these could be staticmethods rather than classmethods, but either is fine by me).

Reply:
Thanks Rick, looks good